tests/cli/test_main.py

```diff
@@ -44,7 +40,7 @@
 def build_config_and_cursor(tmp_path):
     with NamedTemporaryFile(mode='wt', dir=tmp_path) as configfile:
         with NamedTemporaryFile(mode='w+t', dir=tmp_path) as cursorfile:
-            def write_config(config = None):
+            def write_config(config=None):
```

Comment:
This is a tiny unrelated change, but flake8 complained about it and I didn't catch it in the previous PR.